flink-python/pyflink/table/expression.py

```diff
@@ -1618,6 +1618,15 @@ def array_except(self, array) -> 'Expression':
         """
         return _binary_op("arrayExcept")(self, array)
 
+    def array_intersect(self, array) -> 'Expression':
+        """
+        Returns an ARRAY that contains the elements from array1 that are also in array2.
+        If no elements are both in array1 and array2, the function returns an empty ARRAY.
+        If one or both arguments are NULL, the function returns NULL.
+        The order of the elements from array1 is kept.
```

Comment:
this should mention the deduplication behaviour

Reply:
added without duplicates mention like array_union